### PR TITLE
Reduce boilerplate when making API requests

### LINKED ISSUE #1965: Refactor api to remove boilerplate for PUT, POST
Now that https://github.com/mozilla/api.webmaker.org/pull/52 has landed, we can remove some of the boilerplate for PUT and POST responses

### PR DESCRIPTION
**Pull request by gvn**
Closes #1965

### CODE REVIEW

**Comment by gvn** [comment]
Still needs adding/updating elements.

File: www_src/pages/make/make.jsx [hunk 1]
```diff
@@ -69,15 +69,12 @@ var Make = React.createClass({
       if (window.Android) {
         window.Android.setView('/projects/' + body.project.id);
       }
+
+      body.project.author = userInfo;
```
Comment (k88hudson) [inline www_src/pages/make/make.jsx]: interesting, this doesn't get included?

Comment (gvn) [inline www_src/pages/make/make.jsx]: I wasn't seeing it come through in the response...

**Comment by gvn** [comment]
`/element` doesn't seem to have any boilerplate to cut out...

**Comment by gvn** [comment]
I think this is done...

**Comment by thisandagain** [comment]
R+

**Pull request merged**